src/templates/pages/editprefset-addContext-dialog.handlebars

```diff
@@ -0,0 +1,33 @@
+<div class="reveal" id="pmt-dialog-container" data-reveal>
+    <button class="close-button" data-close aria-label="Close modal" type="button">
+        <span aria-hidden="true">&times;</span>
+    </button>
+
+    <h2>{{{messageHelper "gpii_pmt_addContextDialogHeader"}}}</h2>
+    {{{messageHelper "gpii_pmt_addContextDialogInst"}}}
+    <form>
+    <label>Name
+        <input id="pmt-add-context-name-input" type="text" placeholder="preferences01" />
+    </label>
+    {{#if fieldErrors}}
+    <div role="alert" class="alert callout" style="display: block;">
+        <p><i class="fi-alert"></i>{{fieldErrors}}</p>
+    </div>
+    {{/if}}
+
+    <label>Copy initial settings from:
+        <select id="pmt-context-to-copy-select">
+            <option value=""><em>New Blank Set</em></option>
```

Comment:
This text of "New Blank Set" can be from message bundle too? This comment applies to all hardcoded texts in templates.

Reply:
I believe nearly everything should be in message bundles now.